sdk/contentsafety/azure-ai-contentsafety/CHANGELOG.md

```diff
@@ -1,15 +1,40 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0 (Unreleased)
 
 ### Features Added
-
+- Support AAD Authentication
+- Support 8 severity level for text:analyze API
 ### Breaking Changes
+Contract change for AnalyzeText, AnalyzeImage, Blocklist management related methods
+#### AnalyzeText
+- AnalyzeTextOptions
+  - Renamed breakByBlocklists to haltOnBlocklistHit
+  - Add AnalyzeTextOutputType
+- AnalyzeTextResult
+  - Renamed blocklistsMatchResults to blocklistsMatch
+  - Replaced TextAnalyzeSeverityResult by TextCategoriesAnalysis
+#### AnalyzeImage
+- AnalyzeImageOptions
+    - Replaced ImageData by ContentSafetyImageData
+    - Add AnalyzeImageOutputType
+- AnalyzeImageResult
+    - Replaced ImageAnalyzeSeverityResult by ImageCategoriesAnalysis
+#### Blocklist management
+- Added BlocklistAsyncClient
+- Renamed AddBlockItemsOptions to AddOrUpdateTextBlocklistItemsOptions
+- Renamed AddBlockItemsResult to AddOrUpdateTextBlocklistItemsResult
+- Renamed RemoveBlockItemsOptions to RemoveTextBlocklistItemsOptions
+- Renamed TextBlockItemInfo to TextBlocklistItem
 
 ### Bugs Fixed
 
 ### Other Changes
-
+#### Dependency Updates
+- Upgraded `azure-core` from `1.43.0` to version `1.45.0`.
+- Upgraded `azure-core-http-netty` from `1.13.7` to version `1.13.10`.
+- Upgraded `azure-core-test` from `1.20.0` to version `1.22.0`
+- Upgraded `azure-identity` from `1.10.1` to version `1.11.0`
```

Comment:
do not include the test dependencies, customer won't use them.

Reply:
removed test dependency